_config.yml

```diff
@@ -7,25 +7,24 @@ description: > # this means to ignore newlines until "twitter_username:"
   Google search results) and in your feed.xml site description.
 
 # social links
-twitter_username: jekyllrb
-github_username:  jekyll
-rss: rss
-# dribbble_username: jekyll
-# facebook_username: jekyll
-# flickr_username: jekyll
-# instagram_username: jekyll
-# linkedin_username: jekyll
-# pinterest_username: jekyll
-# youtube_username: jekyll
-# googleplus_username: +jekyll
-
-# Mastodon instances
-# mastodon:
-# - username: jekyll
-#   instance: example.com
-# - username: jekyll2
-#   instance: example.com
+social:
+  # mastodon:
+  # - username: jekyll
+  # instance: example.com
```

Comment:
`instance` should be indented two more spaces.

Reply:
Fixed: e80011158bcce15c099c9b7927a967ec8e59d035